Readme changes (#142)
cubing/cubing.js: README.md

<a href="https://alpha.twizzle.net/"><img src="./src/sites/alpha.twizzle.net/twizzle-social-media-image.png" width="256">

Twizzle</a> is the spiritual successor to [alg.cubing.net](https://alg.cubing.net/), based on `cubing.js`. It is currently being developed at [src/sites/alpha.twizzle.net](./src/sites/alpha.twizzle.net/).

> See the [Twizzle Diaries](https://www.youtube.com/watch?v=9_kqXn0Mq-o&list=PLFh3NgpDbzN4VkcfjEZSQ_TYQv_OEjbjF) video series for more information on Twizzle's vision and use cases.

Reviewer: Good call, thanks!

## Usage

> If you would like to use cubing.js as a library in your projects, make sure you have [node](node.js) installed.

Reviewer: Would you mind copying the `node and npm.` phrase and links from below?

With Node installed, run the following command in your terminal:

```shell
npm install cubing
```

> 

Reviewer: I've been holding off on adding this because:

1) the library is not quite stable enough in some places, and
2) if someone doesn't already know how to use `node` and `npm`, we should steer them to using `cdn.cubing.net` directly in a web page.

Hopefully 1) will become less of a problem with documentation.

Would you mind moving this under the current `Documentation` section and renaming them something like this?

- `Documentation` → `Getting started`
- `Usage` → `Using `

If we're showing how to install `cubing`, I think we should also provide some very basic code snippets like "Using `npm`" at https://js.cubing.net/cubing/#javascript

Eventually I'd like to make some videos on how to do this more easily, but for now I'd like to emphasize "if you're already comfortable writing web apps using `npm`, here's how; if you don't already know, you should definitely start with `cdn.cubing.net`".

Reviewer 2: Ok, let me know how the new sections look. They probably need another revision, but hopefully they're closer to the mark. "Documentation" → "Getting started and "Usage" → "Using with Node.js", on the assumption that only people who know they're looking to use it in a Node project will be attracted there. 

I just copy-pasted the import statements from https://js.cubing.net/cubing/#javascript as code snippets - let me know if you want me to (try) add something more substantial.

<twisty-player alg="R U R' U R U2' R'"></twisty-player>
```

You can find more documentation at [js.cubing.net/cubing/](https://js.cubing.net/cubing).  
The source for the documentation site is currently inside the [docs](./docs/) folder of this repository.

## Contributing

If you would like to contribute to the development of `cubing.js`, please refer to our [contribution guidelines](CONTRIBUTING.md).

> Development relies on [node](https://nodejs.org/en/) and [npm](https://docs.npmjs.com/getting-started) (installing `node` will install `npm` as well). Once you have installed those, you can run:

Reviewer: Hmm, the `npm` docs here aren't really that great. I should probably have left them out. How's this?

```suggestion
To work on the `cubing.js` source code, you'll need to install [`node` (and `npm`)](https://nodejs.org/en/). Once you have installed those, you can run:
```

Reviewer 2: I agree the npm docs suck xD From a beginner perspective, I like the inclusion of "(installing `node` will install `npm` as well)", as that took me a second to figure out, and seems to be such ingrained knowledge in the community that it took me a surprisingly amount of Googling to confirm that.

Reviewer: Gonna change this to the middle:

```suggestion
To work on the `cubing.js` source code, you'll need to install [`node`](https://nodejs.org/en/) and [npm](https://docs.npmjs.com/getting-started) (installing `node` will install `npm` as well). Once you have installed those, you can run:
```

This should run `npm install` and then open [the `sites` root file](./src/sites/index.html) at http://localhost:3333/ automatically. Any source code changes will refresh the browser. We don't yet have a development guide, so you'll have to search the code to find where features are implemented.

> The core library code for `cubing.js` is in [src/cubing](./src/cubing/) To test changes, use:

Reviewer: Please keep the code formatting! :-D

(I also just noticed I missed a period right after.)

```suggestion
The core library code for `cubing.js` is in [`src/cubing`](./src/cubing/). To test changes, use:
```
